core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -416,29 +416,32 @@ oppia.controller('CreatorDashboard', [
                 reviewMessage: $scope.reviewMessage
               });
             };
-            $scope.editSuggestion = function(){
-              $scope.showSuggestionEditor = true;
+            $scope.editSuggestion = function() {
+              $scope.suggestionEditorIsShown = true;
             };
             $scope.cancelReview = function() {
               $uibModalInstance.dismiss();
             };
-            $scope.showEditButton = function(){
-              return (!$scope.isNotHandled && $scope.isSuggestionRejected &&
-                !$scope.showSuggestionEditor);
+            $scope.isEditButtonShown = function() {
+              return (
+                !$scope.isNotHandled && $scope.isSuggestionRejected &&
+                !$scope.suggestionEditorIsShown);
             };
-            $scope.showResubmitButton = function(){
-              return (!$scope.isNotHandled && $scope.isSuggestionRejected &&
-                $scope.showSuggestionEditor);
+            $scope.isResubmitButtonShown = function() {
```

Comment:
The show* naming suggests that the method causes the resubmit button to display. However, it's really checking whether the button is shown or not -- so, start with "is".